src/main/java/com/google/neighborgood/servlets/EditTaskServlet.java

```diff
@@ -20,44 +20,25 @@
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that handles the request for editing the details and rewards for a certain task. */
+/** Servlet that handles the request for editing tasks. */
```

Comment:
In my PR#21, I made some changes to this file. Thus, you may need to merge my newly-added code with this file.

Reply:
I'll leave this as is for now, since this branch is branching off Mafe-sps-to-neighborgood which branches from maste, and PR#21 hasn't been merged to master yet. But once it does I'll pull the new changes from master and merge them into this branch as well :)

Reply:
Sounds good!